server/athenian/api/internal/miners/jira/issue.py

```diff
@@ -769,6 +838,17 @@ async def query_jira_raw(
     if jira_filter.status_categories:
         and_filters.append(Status.category_name.in_(jira_filter.status_categories))
 
+    if mapped_to_prs is not None:
+        assert meta_ids
+        PRIssues = NodePullRequestJiraIssues
+        subselect = sa.exists(1).where(
```

Comment:
```suggestion
        subselect = sa.exists().where(
```
this should work

Reply:
yes it's select * instead of select 1, it seems with exists has the same performance